packages/dd-trace/src/plugins/pluginList.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
```

Comment:
This file should be named `plugin_list.js`, although it's probably just not needed at all.